apps/network-alerts/js/notification/notification.js

```diff
@@ -18,12 +18,17 @@ function onNotification(message) {
     return;
   }
 
-  var title = Utils.parseParams(message.imageURL).titleID;
+  var title = null;
+  if (message.data) {
+    title = message.data.title;
+  }
+  else {
+    title = Utils.parseParams(message.imageURL).titleID;
+  }
 
   var url = [
-    'attention.html?title=',
```

Comment:
__issue:__ I believe you should not remove this URL parameter, it's passed to ```attention.html``` -> ```attention.js``` and then used to display emergency alert title - you can see how it look like when you open network alert once again via tapping on notification in notification tray.